iothub/service/src/Common/TimeoutHelper.cs

```diff
@@ -1,19 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
```

Comment:
Can we get rid of TimeoutHelper?

Reply:
We will, once we get rid of all the API flavors that use it.